tests/MyGame/Example/Monster.py

```diff
@@ -15,6 +15,14 @@ def GetRootAsMonster(cls, buf, offset):
         x.Init(buf, n + offset)
         return x
 
+    @classmethod
+    def GetSizePrefixedRootAsMonster(cls, buf, offset):
+        return cls.GetRootAsMonster(buf, offset + flatbuffers.number_types.Int32Flags.bytewidth)
+
+    @classmethod
+    def GetSizePrefix(cls, buf, offset):
```

Comment:
Why is this a generated function? It is identical for all, so could be a global function elsewhere.
Actually, even `GetSizePrefixedRootAsMonster` is not necessary, you could instead just call `GetRootAsMonster(GetBufFromSizePrefixed(buf))` or something, especially since this use case is not as common.

Reply:
Will do!